Kiss.Bff.EndToEndTest/NieuwsEnWerkInstructies.cs

```diff
@@ -180,36 +181,87 @@ public async Task Als_ik_een_oud_bericht_update_komt_deze_bovenaan()
         }
     }
 
+    // 9. Publiceer een bericht met markering Belangrijk 
+    [TestMethod]
+    public async Task Als_ik_een_belangrijk_bericht_publiceer_komt_deze_bovenaan()
+    {
+        var titel = $"End to end test {Guid.NewGuid()}";
+        int initialFeatureCount = 0; // Initialize count
 
-    //[TestMethod]
-    //public async Task Als_ik_een_belangrijk_bericht_publiceer_komt_deze_bovenaan()
-    //{
-    //    var titel = $"End to end test {Guid.NewGuid()}";
-    //    var featuredIndicator = Page.Locator(".featured-indicator");
-    //    var intitialFeatureCount = await featuredIndicator.IsVisibleAsync()
-    //        && int.TryParse(await featuredIndicator.TextContentAsync(), out var c)
-    //            ? c
-    //            : 0;
+        // Declare featuredIndicator outside the try block so it's accessible throughout the method
+        var featuredIndicator = Page.Locator(".featured-indicator");
 
-    //    await CreateBericht(titel, true, "");
+        try
+        {
+            // Step 1: Get the initial featured indicator count (if visible)
+            if (await featuredIndicator.IsVisibleAsync())
+            {
+                var featureText = await featuredIndicator.TextContentAsync();
+                if (int.TryParse(featureText, out var count))
+                {
+                    initialFeatureCount = count;
+                }
+            }
+        }
+        catch (Exception ex)
+        {
+            Console.WriteLine($"Error fetching initial featured count: {ex.Message}");
+        }
+
+        // Step 2: Create a new important message
+        await CreateBericht(titel, true, "");
+
+        try
+        {
+            // Step 3: Go to the page and ensure the news section is visible
+            await Page.GotoAsync("/");
+
+            await Expect(NieuwsSection).ToBeVisibleAsync();
+
+            // Step 4: Check if the newly created important message appears at the top
+            var firstArticle = NieuwsSection.GetByRole(AriaRole.Article).First;
+            await Expect(firstArticle).ToContainTextAsync(titel);
+            await Expect(firstArticle).ToContainTextAsync("Belangrijk"); // Check for "Belangrijk" tag
```

Comment:
check specifiek op de tag, niet of het woord ergens in het hele article voorkomt

Reply:
is aangepaast